install/image-references

```diff
@@ -26,3 +26,15 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-azure-machine-controllers:v4.0.0
+  - name: baremetal-operator
+    from:
+      kind: DockerImage
+      name: quay.io/metal3-io/baremetal-operator:master
```

Comment:
This image is already building in OpenShift, so you should be able to update it to the openshift pullspec similar to the ones above

Reply:
Some of the images defined in the configmap are missed here. See https://github.com/openshift/machine-api-operator/pull/302/files#r303400877
All this images needs to be ART available before merging

Reply:
Done.